### ISSUE TITLE
Web component: Content Block Card - Video - No label for the modal dialog.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5669]
## Detailed description

On JAWS FF/Voiceover iPhone

Issue:
1. In JAWS FF, There is no label for the modal dialog. However Label is present in react staging
2. In VO iPhone, the dialog role is not identified. Issue present in React staging as well.

Expected:
Provide appropriate label for the dialog using either aria-label or aria-labelledby.

**Comment by ariellalgilmore** [comment]
@Praveen-sr This issue should be fixed. Can you confirm?

**Comment by RobertaJHahn** [comment]
Label is still not read in Web Components Canary.
